scipy/fft/_basic.py

```diff
@@ -1166,6 +1166,18 @@ def rfft2(x, s=None, axes=(-2, -1), norm=None, overwrite_x=False, workers=None,
     This is really just `rfftn` with different default behavior.
     For more details see `rfftn`.
 
+    Examples
+    --------
+    >>> import scipy.fft
+    >>> import numpy as np
+    >>> x = np.meshgrid(np.arange(5), np.arange(5))[1]
+    >>> scipy.fft.rfft2(x)
+    array([[ 50.  +0.j        ,   0.  +0.j        ,   0.  +0.j        ],
+           [-12.5+17.20477401j,   0.  +0.j        ,   0.  +0.j        ],
+           [-12.5 +4.0614962j ,   0.  +0.j        ,   0.  +0.j        ],
+           [-12.5 -4.0614962j ,   0.  +0.j        ,   0.  +0.j        ],
+           [-12.5-17.20477401j,   0.  +0.j        ,   0.  +0.j        ]])
```

Comment:
Do you have any objections to changing the values in a way so integers are generated? E.g.:
```suggestion
    >>> x = np.meshgrid([1, 0, -1, 0], [0, 1, 0, 1])[0]
    >>> scipy.fft.rfft2(x)
    array([[0.+0.j, 8.+0.j, 0.+0.j],
           [0.+0.j, 0.+0.j, 0.+0.j],
           [0.+0.j, 0.+0.j, 0.+0.j],
           [0.+0.j, 0.+0.j, 0.+0.j]])
```